HW_15/task_1.py

```diff
@@ -0,0 +1,44 @@
+# Напишите код, который запускается из командной строки и получает на вход путь до директории на ПК.
+# Соберите информацию о содержимом в виде объектов namedtuple.
+# Каждый объект хранит:
+# * имя файла без расширения или название каталога,
+# * расширение, если это файл,
+# * флаг каталога,
+# * название родительского каталога.
+# В процессе сбора сохраните данные в текстовый файл используя логирование.
+
+import argparse
+from collections import namedtuple
+import logging
+
+
+def file_data(path: str):
+    path = path.split("\\")
+    count = len(path)
+    if count >= 3:
+        *_, dir_1, dir_2, temp = path
+    elif count == 2:
+        dir_1 = "not indicated"
+        dir_2, temp = path
+    else:
+        dir_1 = "not indicated"
+        dir_2 = "not indicated"
+        temp = path
+    try:
+        name, ext = str(temp).split(".")
```

Comment:
Так ведь `temp` уже строкового типа
```suggestion
        name, ext = temp.split(".")
```